invenio_rdm_records/config.py

```diff
@@ -691,3 +691,6 @@ def lock_edit_published_files(service, identity, record=None, draft=None):
 
 RDM_RECORDS_ALLOW_RESTRICTION_AFTER_GRACE_PERIOD = False
 """Whether record access restriction is allowed after the grace period or not."""
+
+RDM_RECORDS_REQUIRE_SECRET_LINKS_EXPIRATION = False
```

Comment:
@max-moser since you mainly developed this feature, are you ok with the naming? Or was there a prefix/pattern used for secret links?